index.html

```diff
@@ -56,7 +55,7 @@ <h3>LET'S CONNECT</h3>
         </section>
       </section>
       <section id="my_work">
-        <section id="p1">
+        <!--<section id="p1">
           <img  class="project_image" src="statics/images/Snapshoot_Portfolio.png" alt="snapshoot image project">
```

Comment:
Please follow the design for the pop-up as the previous reviewer also suggested. Project image width and height do not look good as the image is stretched. Kindly see the screenshot below for reference.
![img](https://user-images.githubusercontent.com/35504149/136376806-713f545a-2738-4b6f-b15a-d3c87c45ed67.png)